daffodil-core/src/main/scala/org/apache/daffodil/dsom/Term.scala

```diff
@@ -396,18 +381,18 @@ trait Term
   final lazy val laterSiblings = ListUtils.tailAfter(allSiblings, this)
   final lazy val laterElementSiblings = laterSiblings.collect { case elt: ElementBase => elt }
 
-  final lazy val laterSiblingsWithinEnclosingElement: Seq[Term] = {
-    enclosingElement.flatMap { ee =>
-      enclosingTerm.map { et =>
-        val eeGroup = ee.complexType.group
-        val res =
-          laterSiblings ++
-            (if (et eq eeGroup) Nil
-            else et.laterSiblingsWithinEnclosingElement)
-        res
-      }
-    }.getOrElse(Nil)
-  }
+  //  final lazy val laterSiblingsWithinEnclosingElement: Seq[Term] = {
+  //    enclosingElement.flatMap { ee =>
```

Comment:
You marked this conversation as resolved but didn't delete the code, keeping intentionally?

Reply:
Odd. I will make sure this code is deleted. .